src/tests/unit/services/ThankYouEmailProcessor.test.ts

```diff
@@ -271,7 +85,7 @@ describe("ThankYouEmailProcessor", () => {
 		it("throws error if F2F session can't be found", async () => {
 			mockF2fService.getSessionByYotiId.mockResolvedValueOnce(undefined);
 
-			await expect(thankYouEmailProcessor.processRequest({ session_id: sessionId, topic: "session_completion" })).rejects.toThrow(expect.objectContaining({
+			await expect(thankYouEmailProcessor.processRequest({ session_id: sessionId, topic: "thank_you_email_requested" })).rejects.toThrow(expect.objectContaining({
```

Comment:
How come the topic on these tests changed?

Reply:
Because it's the ThankYouEmailProcessor it's kicked off by the topic being "thank_you_email_requested" - I think this was erroneously set to "session_completion" before?